Sources/StreamChat/Repositories/AuthenticationRepository.swift

```diff
@@ -365,11 +365,10 @@ class AuthenticationRepository {
             switch result {
             case let .success(newToken) where !newToken.isExpired:
```

Comment:
why do we still keep the check? This can also have opposite effect - if you put the time forward.